eng/pipelines/scripts/Get-Linting-Reports.ps1

```diff
@@ -154,3 +154,7 @@ if (-not (Test-Path "$StagingDirectory/troubleshooting")) {
 
 Compress-Archive -Path $OutputDirectory -DestinationPath "$StagingDirectory/troubleshooting/linting-report.zip"
 Write-Host "##vso[task.setvariable variable=HAS_TROUBLESHOOTING]true"
+
+# Fail the build if any of the reports contain errors.
+Write-Host "Linting errors have been found, download the 'linting-report.zip' artifact to see what was found."
```

Comment:
Can the error message have the full path to the file to make it easier to copy/paste?

Reply:
This gets published as a DevOps artifact, I don't think there's a way to create a link to it at this point.

Reply:
The actual download URLs are too dynamic to hardlink to, but you could link to the artifacts, though I don't see a way to link to a specific path/directory within the artifacts.

```
https://dev.azure.com/azure-sdk/$(System.TeamProject)/_build/results?buildId=$(Build.BuildId)&view=artifacts&pathAsName=false&type=publishedArtifacts
```